apps/studio/components/interfaces/Auth/AuthProvidersForm/AuthProvidersForm.tsx

```diff
@@ -23,6 +24,8 @@ import { ProviderForm } from './ProviderForm'
 
 export const AuthProvidersForm = () => {
   const { ref: projectRef } = useParams()
+  const router = useRouter()
+  const { provider: queryProvider } = router.query
```

Comment:
You can use `useParams()` here to avoid the `as string` below

Reply:
JFYI we can probably just call useParams in ProviderForm as well, no need to pass as a prop below